packages/charts/doc/severityLine.md

```diff
@@ -0,0 +1,23 @@
+# Severity Line Chart
+
+Display a level of severity given a value between 1-Low and 4-Critical severity.
+
+```jsx
+import React from react;
+import { SeverityLine } from '@redhat-cloud-services/frontend-components-charts';
+
+<SeverityLine value={2} title='High Severity'>
```

Comment:
what happens when a really long title is entered?

Reply:
Added in the Tooltip and truncation like we talked about 😁